init.sh

```diff
@@ -37,6 +37,7 @@ if [ "$install_directory" != "." ]; then
         mkdir -p .github/workflows
     fi
     mv "$install_directory/.github/workflows"/* .github/workflows
+    rmdir "$install_directory/.github/workflows"
```

Comment:
This is safer, yes. :)
However, this will still leave the `$install_directory/.github` folder, so we'll need to remove this as well - or maybe there is an option to remove the parent as well if it's empty?